doc/sphinx/changes.rst

```diff
@@ -3191,7 +3191,7 @@ Other changes in 8.10+beta1
     by Maxime Dénès, review by Pierre-Marie Pédrot).
 
   - New variant :tacn:`change_no_check` of :tacn:`change`, usable as a
-    documented replacement of :tacn:`convert_concl_no_check`
```

Comment:
This is an interesting issue with rich text in CHANGES I didn't think about. Shouldn't we have a more robust mechanism to handle this situation? cc @Zimmi48

Reply:
I think what we have is fine.  If you forget to remove the `:tacn:`, you get an error message from Sphinx, so it's easily  discovered.  Indeed, unavoidable in CI.